docs/_templates/sidebar.html

```diff
@@ -0,0 +1,55 @@
+<a class="toggle-sidenav d-block d-md-none" href="#"
```

Comment:
Why do we need this file? I thought this was provided by nengo-sphinx-theme.

Reply:
Line 52 is different, it actually gives us content in the sidebar. Without it there's nothing there. We should make it something that's configurable as part of the theme (see https://github.com/nengo/nengo-sphinx-theme/issues/32 for that issue) but rather than wait for that and a release and everything, we can just override the sidebar template locally.